doc/source/parameters/cfd/linear_solver_control.rst

```diff
@@ -117,6 +117,16 @@ When set to ``extra verbose``, the residual at each iteration of the linear solv
 .. caution:: 
 		Be aware that the setup of the ``amg`` preconditioner is very expensive and does not scale linearly with the size of the matrix. As such, it is generally preferable to minimize the number of assembly of such preconditioner. This can be achieved by using the ``inexact newton`` for the nonlinear solver (see :doc:`non-linear_solver_control`).
 
+* There are two additional parameters that can be used in this subsection that only work for the ``lethe-fluid-matrix-free`` application at the moment. They allow to turn on or off the hessian terms present in the Jacobian and the residual (or right-hand side) of the Navier-Stokes problem:
+
+.. code-block:: text
+
+    set enable hessians in jacobian = true
+    set enable hessians in residual = true
```

Comment:
Is it not highly problematic to remove the hessian in the residual ? I would fear that this would quickly lead to a variational inconsistency.

Reply:
Well, in principle, we could leave the hessians in both the jacobian and the residual of the system operator and disable them completely for the mg operator. Here, we would be solving the exact same problem with a slighlty different preconditioner, which can lead to more iterations but if they are not that expensive, that should not be an issue. As for removing the hessians in the system operator, we would have to test more problems and check. I find it nice to have the option to turn them on and off to experiment a bit. We could maybe not write them in the documentation and test them further until we are really sure about their behavior